ideation/README.md

```diff
@@ -1,4 +1,17 @@
 # How to come up with an idea
 
+Sometimes coming up with an idea for a hackathon can be really difficult. However, the good thing to keep in mind is that this is a project to be made in a limited amount of time so it doesn't have to be the perfect idea and can be something that you build off of in the future. It's a good opportunity to try and learn new things with little being risked. Because at hackathons you can work on anything that uses technology, like website/app, a video game, a VR/AR app, etc. there are endless options. 
 
+#### Narrowing down topics 
+What helps me is narrowing down some broader category I want to create a project in. Some hackathons will have a theme or categories you can submit too which can help narrow down the ideas. Others will have company sponsored topics that typically have some kind of specific project they are judging for. It can be helpful to look over these when you're looking for an idea.
 
+#### Create a pool of ideas
+Once you have narrowed down your topic, I like having everyone just pool in ideas and write them down. Typically this will lead to people jumping off each others ideas and in the end you will have a solid list of ideas. When thinking of ideas, it can be beneficial to think about problems that you have faced or that you know about that you want to address. The best ideas I've seen have typically been aimed at solving some sort of problem whether that be wanting to play songs at the same time with friends or wanting to better pair people up in airplane seats based on personality. 
```

Comment:
```suggestion
Something that a lot a teams struggle with is deciding on a common idea with their team. I like having everyone in my team pool together their ideas and write them down. This also leads to the side benefit of bouncing off the different ideas and compile a solid list. During this phase, it can be beneficial to to think about specific problems that you and/or people you know face. The best ideas I've seen have typically been aimed at solving some sort of problem, whether that be wanting to play songs at the same time with friends or wanting to pair people up in airplane seats based on personality. 
```